docs/manual/external-tools.tex

```diff
@@ -906,44 +906,32 @@
 \end{Verbatim}
 
 \item
-  To use a local version of the Checker Framework that
-  you have built from source (Section~\ref{build-source}):
+  To use a locally-modifed version of the Checker Framework, first run:
 
 \begin{alltt}
-  <dependencies>
-    ... existing <dependency> items ...
+./gradlew deployArtifactsToLocalRepo
+\end{alltt}
 
-    <!-- Annotations from the Checker Framework: nullness, interning, locking, ... -->
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>checker-qual</artifactId>
-      <version>\ReleaseVersion{}</version>
-      <scope>system</scope>
-      <systemPath>\$\{env.CHECKERFRAMEWORK\}/checker/dist/checker-qual.jar</systemPath>
-    </dependency>
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>checker</artifactId>
-      <version>\ReleaseVersion{}</version>
-      <scope>system</scope>
-      <systemPath>\$\{env.CHECKERFRAMEWORK\}/checker/dist/checker.jar</systemPath>
-    </dependency>
-    <!-- The annotated JDK to use. -->
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>jdk8</artifactId>
-      <version>\ReleaseVersion{}</version>
-      <scope>system</scope>
-      <systemPath>\$\{env.CHECKERFRAMEWORK\}/checker/dist/jdk8.jar</systemPath>
-    </dependency>
-    <dependency>
-    <groupId>com.google.errorprone</groupId>
-      <artifactId>javac</artifactId>
-      <version>9+181-r4173-1</version>
-    </dependency>
-  </dependencies>
+\noindent
+Then, in the project you want to type-check, use
+\<\ReleaseVersion{}-SNAPSHOT> as the version number:
+
+\begin{alltt}
+   <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
This seems like the wrong place for these instructions.  This text appears in point 1 in a numbered list, but point 3 in the list is the first one that mentions the `checker` artifact.  I think it should appear in point 3, or more likely in a new subsection that appears at the end of the Maven section.